pages/linux/getconf.md

```diff
@@ -0,0 +1,24 @@
+# getconf
+
+> getconf - get configuration values from your Linux system.
+> 
+
+- List every configuration values available
+
+`getconf -a`
+
+- List the configuration values for a specific directory
+
+`getconf -a {{directory}}` 
```

Comment:
```suggestion
- List the configuration values for a specific directory:

`getconf -a {{path/to/directory}}` 
```

Reply:
Some additions to a suggestion above:

- This change is [mandatory](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths).
- Path format should be standardized across all pages.